doc/_static/mpl.css

```diff
@@ -683,20 +683,64 @@ figcaption {
     outline-color: #003c63;
 }
 
-.flex-container {
-    /* display part copy-pasted from: https://css-tricks.com/using-flexbox/ */
-    display: -webkit-box;      /* OLD - iOS 6-, Safari 3.1-6 */
-    display: -moz-box;         /* OLD - Firefox 19- (buggy but mostly works) */
-    display: -ms-flexbox;      /* TWEENER - IE 10 */
-    display: -webkit-flex;     /* NEW - Chrome */
-    display: flex;             /* NEW, Spec - Opera 12.1, Firefox 20+ */
-
-    flex-wrap: wrap;
-    flex-direction: row;
-    align-content: center;
-    justify-content: center;
-}
-
-.flex-item {
-    margin: 0px;
+
+div.responsive_banner {
+    /* Horizontally centered */
+    display: block;
+    margin: auto;
+
+    max-height: 139px; /* at most 1 x 1:1 subfig height */
+    width=100%;
```

Comment:
What is a mistake? (My wish was to have the banner to take the full column width, with its height being reduced if needed.)